app/src/main/java/co/nimblehq/alpine/sample/nfc/PassportInformationActivity.kt

```diff
@@ -0,0 +1,32 @@
+package co.nimblehq.alpine.sample.nfc
+
+import android.os.Bundle
+import android.view.inputmethod.EditorInfo
+import androidx.activity.ComponentActivity
+import co.nimblehq.alpine.databinding.ActivityPassportInformationBinding
+
+class PassportInformationActivity : ComponentActivity() {
+
+    private val binding: ActivityPassportInformationBinding by lazy {
+        ActivityPassportInformationBinding.inflate(layoutInflater)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        setupView()
+    }
+
+    private fun setupView() {
+        with(binding) {
+            passportInfoPassportNumber.editText?.imeOptions = EditorInfo.IME_ACTION_DONE
```

Comment:
Why don't we put this in the layout file instead? What do you think? 😄

Reply:
Yeah, I agreed will update 👍